package.json

```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "next start -p $PORT",
     "fix:format": "prettier --write \"**/*.{js,json,md}\" --ignore-path .gitignore",
-    "test": "npm-run-all test:*",
+    "test": "jest",
```

Comment:
Wait.. Bryan, is this correct?

Reply:
I fixed this is a later commit

Reply:
The docs suggested that change. But it makes more sense to have a test:jest command